### DISCUSSION TITLE
Loading the model that is downloaded locally on my PC in the run_localGPT_API.py

### DISCUSSION THREAD

**Discussion by Loves4** [PromtEngineer/localGPT#372] [Q&A]
i want to use the downloaded model that i have downloaded from huggingface instead of it downloading when i run the run_localGPT_API.py, i request the community for help

**Comment by bondocai** [comment]
I was able to use a downloaded model on Windows11 by placing the model in this location and adjusting the card name: 
"C:\Users\yourusername\.cache\huggingface\hub\models--TheBloke--Llama-2-7b-Chat-GPTQ"
Note the naming conventions uses double dashes vs space or slashes.  You will also find other models as well in the \hub folder- however, you will have to download the models and replace missing files as needed.

**Reply by bondocai** [reply]
for some reason the backslash dissapears before .cache...

**Comment by abcnow** [comment]
how about if you are doing this on Mac? Any idea?

**Reply by bondocai** [reply]
believe this is the location:
~/.cache/huggingface/hub

**Reply by bondocai** [reply]
also found this site quite useful on how to install huggingface on a Mac:
https://github.com/domschl/HuggingFaceGuidedTourForMac
